src/dashboard/src/pages/Jobs/index.tsx

```diff
@@ -14,17 +15,22 @@ import {
 import {
   Container,
   FormControl,
+  Grid,
   InputLabel,
+  Link,
   Paper,
   Tabs,
   Tab,
+  Tooltip,
   Toolbar
 } from '@material-ui/core'
 import SwipeableViews from 'react-swipeable-views'
 
 import ClustersContext from '../../contexts/Clusters'
 import ClusterSelector from '../../components/ClusterSelector'
 
+import useFetch from 'use-http'
+import { Info } from '@material-ui/icons'
```

Comment:
Put @material-ui/icons at the bottom of @material-ui/core to prevent repeat imports